db/migrate/20230819054513_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
```

Comment:
- [ ] in the user table there should be a column for post_counter according to the requirements

Reply:
- Well resolved